pkg/ebpf/bytecode/asset_windows.go

```diff
@@ -0,0 +1,26 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+// +build windows
+
+package bytecode
+
+import (
+	"fmt"
+	"io"
+)
+
+// AssetReader describes the combination of both io.Reader and io.ReaderAt
+type AssetReader interface {
+	io.Reader
+	io.ReaderAt
+	io.Closer
+}
+
+// VerifyAssetPermissions is for verifying the permissions of bpf programs
+func VerifyAssetPermissions(assetPath string) error {
+	return fmt.Errorf("verification of bpf assets is not supported on windows")
```

Comment:
Is this necessary? There are no ebpf assets on windows

Reply:
It needs to be defined on windows because it's used here: https://github.com/DataDog/datadog-agent/blob/18184094e46350b5b0e5ee8bcd0d4a3a67611890/pkg/ebpf/bytecode/asset_nobindata.go#L21

and there isn't a non-windows build tag protecting that file. Since it's all ebpf-related code, we could add the `!windows` build tag there & wherever that function is used if we want

Reply:
We weren't ever calling `GetReader` from windows, so it didn't matter (and this file didn't exist before and everything was OK). Has that changed?

Reply:
What's changed is that `VerifyAssetPermissions` contains code that doesn't compile on windows: https://github.com/DataDog/datadog-agent/blob/18184094e46350b5b0e5ee8bcd0d4a3a67611890/pkg/ebpf/bytecode/asset.go#L33

So we need a windows version & a non-windows version, even if `GetReader` is never called from windows

Reply:
Can you create an `asset_linux.go` and move the `VerifyAssetPermissions` there? That way you don't have to redefine `AssetReader` here, and can remove the build tags from `asset.go`

Reply:
Sure!